config/environment.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    emberResponsiveDefaults: {
+      autoInject: true
```

Comment:
The way I was originally thinking about this was to pass a set of factories you wanted to inject into like this:

```javascript
emberResponsiveDefaults: {
   autoInject: ['controller']
}
```

I think that kind of strategy is useful if you wanted to support that feature in the future. If the goal is to fully remove this functionality, the way you have it is probably best.